src/mui/MuiMessage.tsx

```diff
@@ -65,26 +75,44 @@ export function MuiMessage({
         justifyContent={message.self ? 'flex-end' : 'flex-start'}
         style={{ overflowWrap: 'break-word' }}
       >
-        {message.avatar && !message.self && ChatAvator}
-        <Box minWidth={0} display="flex" flexDirection="column">
-          {message.username && ChatUsername}
-          <Box
-            maxWidth="100%"
-            py={1}
-            px={2}
-            bgcolor={message.self ? 'primary.main' : 'background.paper'}
-            color={message.self ? 'primary.contrastText' : 'text.primary'}
-            borderRadius={16}
-            boxShadow={2}
-          >
-            {message.type === 'text' && (
-              <Typography variant="body1" style={{ whiteSpace: 'pre-wrap' }}>
-                {message.content}
-              </Typography>
-            )}
-            {message.type === 'jsx' && <div>{message.content}</div>}
-          </Box>
-          {showDateTime && ChatDate}
+        <Box
+          minWidth={0}
+          py={1}
+          px={2}
+          bgcolor={bgcolor}
+          color={color}
+          borderRadius={16}
+          boxShadow={2}
+        >
+          {message.type === 'text' && (
+            <Typography
+              variant="body1"
+              style={{ overflowWrap: 'break-word', whiteSpace: 'pre-wrap' }}
+            >
+              {message.content}
+            </Typography>
+          )}
+          {message.type ==='text_audio' && (
+            <div style={{flex: '0 1 0%', display: 'flex', flexDirection: 'row'}}>
```

Comment:
Should be passed as custom component. You can pass as custom only button for play audio